lib/base/Page.ts

```diff
@@ -0,0 +1,278 @@
+"use strict";
+
+import Version from "./Version";
+import Response from "../http/response";
+import RestException from "./RestException";
+
+export interface TwilioResponsePayload {
+  first_page_uri: string;
+  next_page_uri: string;
+  page: number;
+  page_size: number;
+  previous_page_uri: string;
+  uri: string;
+  meta?: {
+    key?: string;
+    next_page_url?: string;
+    previous_page_url?: string;
+  };
+}
+
+interface Solution {
+  [name: string]: any;
+}
+
+type META_KEYS =
+  | "end"
+  | "first_page_uri"
+  | "last_page_uri"
+  | "next_page_uri"
+  | "num_pages"
+  | "page"
+  | "page_size"
+  | "previous_page_uri"
+  | "start"
+  | "total"
+  | "uri";
```

Comment:
Is declaring type `META_KEYS` necessary? Since its being declared again in L80-92

Reply:
I don't think that the `META_KEYS` type alias is necessary, but I think it provides the user with more useful information about what the `Page.META_KEYS` static property could be: a list of `META_KEYS` values rather than simply a list of `string` values.